traffic_portal/app/src/common/modules/form/user/form.user.tpl.html

```diff
@@ -82,6 +82,14 @@
                     <small ng-show="user.tenantId"><a href="/#!/tenants/{{user.tenantId}}" target="_blank">View Details&nbsp;&nbsp;<i class="fa fs-xs fa-external-link"></i></a></small>
                 </div>
             </div>
+            <div class="form-group" ng-class="{'has-error': hasError(userForm.uCDN), 'has-feedback': hasError(userForm.uCDN)}">
+                <label for="uCDN" class="control-label col-md-2 col-sm-2 col-xs-12">Upstream CDN</label>
+                <div id="uCDN" class="col-md-10 col-sm-10 col-xs-12">
```

Comment:
`div` is not a [label-able element](https://html.spec.whatwg.org/multipage/forms.html#category-label), the `id` that corresponds to the `label`'s `for` attribute should be on the control it labels.